scripts/build.sh

```diff
@@ -31,10 +31,11 @@ elif [[ $FLAVOR == "cli" ]]; then
     build_container cli
 else
     build_container cli
-    if [ $? eq 1 ]; then
-	exit 1
+    if [ $? -ne 0 ]; then
+    	echo "Building container failed. Exiting"
+	return 1
     fi
     build_container novnc
 fi
 
-exit $?
+return $?
```

Comment:
This change breaks the build.sh scripts:

```
Built rgardler/simdem_novnc:0.7.4-dev
./scripts/build.sh: line 41: return: can only `return' from a function or sourced script
```

This is caught by the pre-commit.sh script - [please enable it on your dev machine](https://github.com/Azure/simdem#contributing).

I'm going to change it back to exit - if there is a good reason why it needs to be return then please open an issue.